packages/@aws-cdk/cdk/lib/physical-name-generator.ts

```diff
@@ -0,0 +1,75 @@
+import crypto = require('crypto');
+import { IResource } from "./resource";
+import { Stack } from './stack';
+
+export class PhysicalNameGenerator {
```

Comment:
Turn this into a simple function. I don't see the need for a whole class:

```ts
export function generatePhysicalName(resource: IResource): string;
```

Reply:
Done.